### ISSUE TITLE
[Bug Report] "v" and "c" characters cannot be typed into the text field in some website when enabling this extension in Firefox

### ISSUE THREAD

**Issue by c02y** [Mr-VincentW/PhotoShow#187]
This extension is very useful, thanks for your work.

But recently I noticed that when I try to write some text in text field of reddit.com, `s` and `c` are disabled, typing them does nothing, after disabling the extension, those two characters are working fine again.

I didn't test all the shortcuts, but since the two characts are hard coded in the extension:
```
S : Save high-definition image
C: Copy high-definition image address
```

maybe providing a disable-all-builtin-shortcuts function would be better, I personally never use any shortcut from the extension.

FYI:
Firefox: 131.0.2
PhotoShow: 4.48.0
OS: Archlinux

-------------------------------------------------------
**UPDATE**:
My bad: I see no Preference fucntion in the Mozilla Add-ons Manager page, thinking that there is no setting for the extension, it turns out I have to left click the icon in the tool bar of Firefox.
![image](https://github.com/user-attachments/assets/3ad80bf3-7135-47c2-9002-daacbb716a33)

But the thing is, after I uncheck all those shortcuts, I click the icon again, they are all restored.

**Comment by garyking** [comment]
A similar issue is that the letter "C" used for keyboard shortcuts, such as to enable and disable captions on YouTube videos, does not work when this extension is enabled.

**Comment by Mr-VincentW** [comment]
> But recently I noticed that when I try to write some text in text field of reddit.com, `s` and `c` are disabled, typing them does nothing, after disabling the extension, those two characters are working fine again.

Hi, thanks for bringing this to my attension! It has been fixed in update 4.49. Please kindly wait for the new version to get available in each browser's app store after the review process.



> But the thing is, after I uncheck all those shortcuts, I click the icon again, they are all restored.

Technically, PhotoShow should retain its settings after the popup closes. According to extension development APIs, these settings are saved with your browser profile, or locally if no profile is set up.

Unfortunately, I don’t have a Linux environment to test this issue. If you haven’t already, try setting up a browser profile to see if that helps.

I’ll close this issue for now, but feel free to reopen it if these issue persist. Thanks!

**Comment by c02y** [comment]
Yes, I just tried the new version, the "s" and "c" keys are working fine in the text field, and the popup window preference window when I uncheck/check some options is working normally as well. Thank you.
